compiler/src/dotty/tools/dotc/transform/Splicer.scala

```diff
@@ -113,14 +113,22 @@ object Splicer {
     }
 
     protected def interpretStaticMethodCall(moduleClass: Symbol, fn: Symbol, args: => List[Object])(implicit env: Env): Object = {
-      val instance = loadModule(moduleClass)
+      val (instance, clazz) =
+        if (moduleClass.name.startsWith(str.REPL_SESSION_LINE)) {
+          (null, loadReplLineClass(moduleClass))
+        } else {
+          val instance = loadModule(moduleClass)
+          (instance, instance.getClass)
+        }
+
```

Comment:
Do we need `instance` in the return? Seems it is not used.

Reply:
It is used in the last line of the method. https://github.com/lampepfl/dotty/pull/5688/files/b54e3c586d190fbb02df0e9f5b3b3c13a0c3c5cc#diff-43be0f2f8a9625bb70763569c65a6fdcR132